release/windows/choco/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,45 @@
+﻿$ErrorActionPreference = 'Stop';
+
+$toolsDir   = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$url        = '{{DownloadUrl}}'
```

Comment:
where is DownloadUrl defined? Is url unused because we only support 64bit on Windows?

Reply:
Yes, I believe so.  Should we add support for 32bit as well?